feat(tokens): provide autofix for all global tokens (#763)
patternfly/pf-codemods: packages/eslint-plugin-pf-codemods/src/rules/v6/tokensUpdate/tokens-update.md

### tokens-update

We have updated our CSS tokens. About half of our tokens have been replaced with newer ones. 

- this rule provides an autofix for global non color tokens

> - global color tokens will be replaced with a temporary hot pink color token (`t_temp_dev_tbd` react token or `--pf-t--temp--dev--tbd` class)
>   - don't forget to replace these tokens later

Reviewer: ```suggestion
- global color tokens will be replaced with a temporary hot pink color token **that must be manually replaced** (`t_temp_dev_tbd` react token or `--pf-t--temp--dev--tbd` CSS variable)
```

Slight tweak to make needing to replace this token stand out a little more
